xbmc/interfaces/json-rpc/schema/version.txt

```diff
@@ -1 +1 @@
-JSONRPC_VERSION 12.7.0
```

Comment:
only for info why version 12.8 has been skipped?

Reply:
Ah, my bad!  Skipped 1 too far when rebasing I guess :)  Thanks for spotting it.